tooling/minimize/tests/pass/slice.rs

```diff
@@ -0,0 +1,15 @@
+// Pass slice reference accross function boundaries
+pub fn assert_some_elements(a: &[i32]) {
+    assert!(a[1] == -40);
+    assert!(a[2] == 30);
+    assert!(a[3] == -20);
+}
+
+fn main() {
+    let x: [i32; 5] = [50, -40, 30, -20, 10];
+    let slice: &[i32] = &x;
```

Comment:
Can you also add a variant that uses `slice::from_raw_parts`? Or do you want to make that a separate PR?
(It requires another new case in minimize to support `aggregate_raw_ptr`.)